lib/dialogs/trash-tag-confirmation/index.tsx

```diff
@@ -28,7 +28,9 @@ const TrashTagConfirmation: FunctionComponent<Props> = ({
     onDone={closeDialog}
     title="Delete Tag"
   >
-    <p>Are you sure you want to delete &quot;{tagName}&quot;?</p>
+    <div className="dialog-inner-content">
+      Are you sure you want to delete &quot;{tagName}&quot;?
```

Comment:
if it's giving you a headache with the formatting you can also produce the entire string in JavaScript

```ts
<div className="dialog-inner-content">{`Are you sure you want to delete '${tagName}'?`</div>
```

strings like this will be properly escaped by React for HTML

Reply:
Thanks. For once this is something I learned already :-)
The changes here were more about getting the spacing to be consistent with the designs.
I used the `&quot;` characters following another example I had found in the codebase.
Do you prefer one way over the other?

Reply:
no preference from me!